CHANGELOG.md

```diff
@@ -1,3 +1,108 @@
+## 1.15.0 (2019-10-02)
+
+This feature release adds secure support for Git over HTTPS, updates
+`kubectl` and `kustomize`, de-experimental-ises garbage collection,
```

Comment:
I like "de-experimenal-ises" as a bit of word play, but I think we should provide a plain meaning either here or in the latter use. This might be easier if this bit is split into its own sentence: "Garbage collection, namespace scoping, and manifest generation are no longer considered experimental".